client/my-sites/site-settings/style.scss

```diff
@@ -406,21 +406,18 @@
 	margin: 16px 36px 0;
 }
 
-.site-settings__date-time-format-custom {
-	.form-radio {
-		display: none;
-		margin-top: 12px;
-	}
-	&.is-custom .form-radio {
-		display: block;
-	}
+.site-settings__date-time-format {
+	margin-top: -16px;
 
-	.form-text-input {
-		margin: 0 12px;
-		width: 100px;
+	.site-settings__date-time-format-title {
+		color: $gray-dark;
+		font-size: 16px;
+		line-height: 24px;
+		padding-bottom: 10px;
 	}
-
-	.site-settings__date-time-format-custom-preview {
-		white-space: nowrap;
+	.site-settings__date-time-format-info {
+		color: $gray;
```

Comment:
Done!
I've also replaced `$gray-dark` with `$gray-text` as, while they are the same color, it makes more sense to me to use `-text` for a text color.